source/_patterns/basics/dropdowns/00-dropdown.hbs

```diff
@@ -1,7 +1,11 @@
-<div class="dropdown {{ dropdownState }} {{#if pullright}}pull-right{{/if}}" aria-labelledby="{{ dropdownAriaLabelledby }}">
-  <button type="button" class="btn {{ btnClass }} dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-    {{ dropdownText }}
-  </button>
+<div class="dropdown {{ dropdownState }} {{#if isKebab}}pf-dropdown__kebab{{/if}} {{#if pullright}}pull-right{{/if}}" aria-labelledby="{{ dropdownAriaLabelledby }}">
+  {{#if isKebab}}
+    {{> basics-icon-button btnText="" btnClass="btn-link" iconFamily="fa" iconName="fa-ellipsis-v" iconModifier="" ariaLabeledBy="Kebab"}}
+  {{else}}
```

Comment:
I included .pf-dropdown-kebab as a way to style the kebab variation of dropdown differently than the dropdown in the context of the card. I'm open to suggestions on this. Ultimately the issue is that visual right edge of the kebab and the actual right edge of the kebab are not the same (whereas in the dropdown they are), so to get the kebab to visually line up in the same position as the dropdown, I need to be able to target it separately and give it some negative margin to push it over (look at these examples of the [kebab](https://jgiardino.github.io/patternfly-atomic/?p=components-card-header-kebab) and [dropdown](https://jgiardino.github.io/patternfly-atomic/?p=components-card-header-dropdown))